shipitfile.js

```diff
@@ -0,0 +1,33 @@
+module.exports = (shipit) => {
+  require('shipit-deploy')(shipit);
+
+  shipit.initConfig({
+    default: {
+      workspace: '.',
+      deployTo: process.env.DEPLOY_SERVICE_PATH,
+      // don't send node_modules of packages since they've been bundled,
+      // send root node_modules since we need them to start the app.
+      ignores: ['packages/**/node_modules'],
+      keepReleases: 5,
+      keepWorkspace: true,
+      shallowClone: false,
+      branch: 'HEAD',
```

Comment:
We can keep it just for easing testing on a feature branch. But CircleCI should always stick with the 'main' branch for the production deployment.

Reply:
HEAD is for not having to repeat main, such as in the DRY principles, in case branch named may change, the circle ci config will choose the correct branch, while here HEAD will do.